vision_datasets/image_classification/classification_as_kvp_dataset.py

```diff
@@ -70,20 +55,79 @@ def __init__(self, classification_dataset: VisionDataset):
         super().__init__(dataset_info, dataset_manifest, dataset_resources=classification_dataset.dataset_resources)
 
     def construct_schema(self, class_names: typing.List[str]) -> typing.Dict[str, typing.Any]:
-        schema: typing.Dict[str, typing.Any] = BASE_CLASSIFICATION_SCHEMA  # initialize with base schema
-        schema["fieldSchema"][f"{CLASS_NAME_KEY}"]["classes"] = {c: {"description": f"A single class name. Only output {c} as the class name if present."} for c in class_names}
+        raise NotImplementedError
+
+    def construct_kvp_label_data(self, label_names: typing.List[str]) -> typing.Dict[str, typing.Union[typing.Dict[str, typing.Dict[str, str]], None]]:
+        raise NotImplementedError
+
+
+class MultiClassAsKeyValuePairDataset(ClassificationAsKeyValuePairDatasetBase):
```

Comment:
Nit: maybe `MulticlassClassificationAsKeyValuePairDataset`? Same for `MultilabelClassificationAsKeyValuePairDataset` (for clarity, outside of the context of `ClassificationAsKeyValuePairDatasetBase`).